scripts/run_backend_tests_test.py

```diff
@@ -916,8 +950,92 @@ def mock_subprocess_run(cmd: List[str], **_: str) -> MockProcessOutput:
             returned_output, coverage = run_backend_tests.check_coverage(
                 False, data_file=data_file)
 
-        self.assertEqual(returned_output, coverage_report_output)
-        self.assertEqual(coverage, 86)
+        expected_output = expected_coverage_report.strip()
```

Comment:
Here and elsewhere, why do you need the strip()? Can you just update the string constants so that they can be used as-is?

Reply:
@seanlip, the utilization of the strip() method within this context serves to eliminate leading and trailing whitespace characters from the strings. The discrepancy observed between the expected output and the output obtained from the check_coverage function is attributed to the presence of whitespace, thus necessitating the inclusion of .strip(). This ensures consistency and accuracy in the comparison process by disregarding extraneous whitespace.

Reply:
Well, why not drop the extraneous whitespace from the expected string? You control that string completely, after all.

Reply:
@seanlip now removed the .split() from test code.